src/modules/view-all/ViewAll.tsx

```diff
@@ -0,0 +1,12 @@
+
+
+const ViewAll = () => {
+    return
+
+    return (
+        <></>
+    )
+
+}
+
```

Comment:
Are these empty components created for future use? this one and ListOfAllItems

Reply:
Yes, that's right. The plan was to implement them in a different branch as they are not necessarily part of the home page